docs/installation/server-config/index.rst

```diff
@@ -83,16 +78,18 @@ Here are some common configuration properties that you might want to customize.
 .. _ksql-auto-offset-reset:
 
 ^^^^^^^^^^^^^^^^^
-auto.offset.reset
+ksql.streams.auto.offset.reset
```

Comment:
Please ensure proper .rst syntax.  The `^^^^^^^` lines are too short.

Reply:
👍 I didn't know they had to match the length of the text. How weird!